### PR TITLE
Styles the show/work page.

### PR DESCRIPTION
**Pull request by bwatson78**
1. lux.scss: adds two new partials to the main style page.
2. _cards.scss: changes border radius to 0 and adds size to font.
3. _show_button.scss: refines styling for the buttons on main show block.
4. _show_heading.scss: corrects the font styling on show header.
5. show.html.erb: changes classes away from bootstrap standard to EUL standard.

### CODE REVIEW

File: app/views/catalog/_metadata_block.html.erb [hunk 1]
```diff
@@ -6,8 +6,10 @@
     <div class="card-body">
       <dl>
         <% fields.each do |field_name, field| %>
-          <dt class="blacklight-<%= field_name.parameterize %>"><%= render_document_show_field_label document, field: field_name %></dt>
-          <dd class="blacklight-<%= field_name.parameterize %>"><%= doc_presenter.field_value field %></dd>
+          <div class="card-line">
```
Comment (wulman16) [inline app/views/catalog/_metadata_block.html.erb:9]: `card-line` could be changed to be Bootstrap's `row` class. There are certain exception metadata blocks that we've rerouted away from this generic metadata block file (`_access_and_copyright_block.html.erb`, for example), so can you make sure that all blocks have been changed?

File: app/assets/stylesheets/lux/_show_heading.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@import "lux/variables";
+
+div#document.document > div > h1 {
```
Comment (wulman16) [inline app/assets/stylesheets/lux/_show_heading.scss:3]: Could you add a class to the element that this is referring to and define the rule that way?

File: app/assets/stylesheets/lux/_show_heading.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  font: normal bold 48px/normal "Cardo", serif;
```
Comment (wulman16) [inline app/assets/stylesheets/lux/_show_heading.scss:4]: Could you adapt one of our variables in `_fonts.scss` to achieve this, with rems?

File: app/assets/stylesheets/lux/_show_button.scss [hunk 4]
```diff
@@ -0,0 +1,46 @@
+@import "lux/variables";
```
Comment (wulman16) [inline app/assets/stylesheets/lux/_show_button.scss:1]: Could you please convert the larger pixel values in this file to use rems?

File: app/assets/stylesheets/lux/_cards.scss [hunk 5]
```diff
@@ -3,7 +3,31 @@
 div.card-header {
   font-weight: $font-weight-bold;
   text-transform: uppercase;
-  font-size: $font-size-sm;
+  font-size: .80rem;
   background-color: $dark-blue;
   color: $gold;
 }
+
+div.card-header:first-child {
```
[the remark this answers is not in the document]
Comment (bwatson78) [reply app/assets/stylesheets/lux/_cards.scss:11]: This has been addressed and in the next commit.

File: app/assets/stylesheets/lux/_cards.scss [hunk 6]
```diff
@@ -3,7 +3,31 @@
...
+  border-radius: 0;
+}
+
+@media (min-width: 768px) {
```
[the remark this answers is not in the document]
Comment (bwatson78) [reply app/assets/stylesheets/lux/_cards.scss:15]: This has been addressed and in the next commit.

File: app/assets/stylesheets/lux/_show_heading.scss (hunk 2 above)
Comment (bwatson78) [reply app/assets/stylesheets/lux/_show_heading.scss:3]: This has been addressed and in the next commit.

File: app/views/catalog/_metadata_block.html.erb (hunk 1 above)
Comment (bwatson78) [reply app/views/catalog/_metadata_block.html.erb:9]: This has been addressed and in the next commit.

File: app/assets/stylesheets/lux/_show_heading.scss (hunk 3 above)
Comment (bwatson78) [reply app/assets/stylesheets/lux/_show_heading.scss:4]: This has been addressed and in the next commit.

File: app/assets/stylesheets/lux/_show_button.scss (hunk 4 above)
Comment (bwatson78) [reply app/assets/stylesheets/lux/_show_button.scss:1]: This has been addressed and in the next commit.

**Comment by bwatson78** [comment]
Screen Shot 2020-04-03 at 8 28 41 AM

**Comment by bwatson78** [comment]
Screen Shot 2020-04-06 at 2 49 24 PM

**Comment by bwatson78** [comment]
Screen Shot 2020-04-06 at 2 47 45 PM

**Pull request merged**